app/models/green_lanes/goods_nomenclature.rb

```diff
@@ -21,6 +21,8 @@ def filter_by_category(category)
     if applicable_category_assessments.any?
       grouped = applicable_category_assessments.group_by(&:category)
       grouped[category]
+    else
+      []
```

Comment:
a simpler approach to achieve the same thing would be as follows?

```ruby
grouped = applicable_category_assessments.group_by(&:category)
grouped[category] || []
```

Reply:
Done